Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp

```diff
@@ -380,6 +381,29 @@ static void RunAssetScan(
 			FObjectPropertyBase::GetExportPath(Class.Get(), nullptr /*Parent*/, nullptr /*ExportRootScope*/, 0 /*PortFlags*/));
 		});
 
+	// Take account of any core redirects for the blueprint classes we want to scan.
+	auto FilterArray = Filter.TagsAndValues.Array();
+	for (auto OneEntry : FilterArray)
+	{
+		if (OneEntry.Value.IsSet())
+		{
+			// I don't know what this CoreUObject wrapper is, but it needs removing and then adding back to any previous name.
+			FString ClassName = OneEntry.Value.GetValue();
+			ClassName.RemoveFromStart("/Script/CoreUObject.Class'");
```

Comment:
It might be more efficient to iterate over the `FilterBaseClasses` array instead of the TagsAndValues map, because the elements are the `UClass` pointers and then use the `::GetFName` method to retrieve the simple name that is needed for the `FindPreviousNames` call.

Reply:
Yep a good improvement! Have submitted a change with this.
Note that I'm still unsure about using this string "/Script/CoreUObject.Class'" - but it is hardcoded in parts of the UE source too ...
I did find an alternate way to add it using FObjectPropertyBase::GetExportPath() but that relies on using the non-redirected class which I'm not sure makes sense. But I've left a comment in the code with that version in case it is useful.

Reply:
And with the change I've checked the generated arrays are still the same as before + it still works in my project.